bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -193,6 +197,26 @@ public Connection(@Nullable Connection oldConnection, Gson gson) {
         gsonWithNullSerialization = gsonBuilder.create();
     }
 
+    /**
+     * Check if deviceId is valid (consisting of hex(hex(16 random bytes)) + correct ending)
+     *
+     * @param deviceId the deviceId
+     * @return true if valid, false if invalid
+     */
+    private boolean checkDeviceIdIsValid(@Nullable String deviceId) {
+        if (deviceId != null && deviceId.length() == 92 && deviceId.endsWith("23413249564c5635564d32573831")) {
+            try {
```

Comment:
https://github.com/openhab/openhab-addons/pull/8211 my version do the check without throwing and catching exceptions